checkov/arm/checks/resource/FunctionAppHttpVersionLatest.py

```diff
@@ -0,0 +1,22 @@
+from checkov.arm.base_resource_value_check import BaseResourceValueCheck
+from checkov.common.models.enums import CheckCategories
+
+
+class FunctionAppHttpVersionLatest(BaseResourceValueCheck):
+
+    def __init__(self) -> None:
+        name = "Ensure that 'HTTP Version' is the latest, if used to run the Function app"
+        id = "CKV_AZURE_67"
+        supported_resources = ("Microsoft.Web/sites/slots",)
```

Comment:
```suggestion
        supported_resources = ("Microsoft.Web/sites/slots", "Microsoft.Web/sites")
```

Sites can also have this set: https://learn.microsoft.com/en-us/azure/templates/microsoft.web/sites?pivots=deployment-language-arm-template#siteconfig-1

Can you add a UT for this?